### PR TITLE
Rename "group name" into "cluster name"

### PR DESCRIPTION
**Pull request by petrpleshachkov**

### CODE REVIEW

**Comment by Serdaro** [comment]
@petrpleshachkov , Any idea why the build is not successful? Is it caused by 4.0-SNAPSHOT being not updated?

**Comment by petrpleshachkov** [comment]
@Serdaro , yes, we have to use the newest hazelcast master to compile.

File: src/docs/asciidoc/setting_up_clusters.adoc [hunk 1]
```diff
@@ -336,26 +336,26 @@ so that you can isolate your clusters.
 * `port`: Integer value that is used to set the multicast port.
 
 
-[[creating-cluster-groups]]
-=== Creating Cluster Groups
+[[creating-cluster]]
+=== Creating Cluster
```
Comment (Serdaro) [inline src/docs/asciidoc/setting_up_clusters.adoc]: Let's keep the [creating-cluster-groups] anchor as it is, not to get 404s for the existing links to this section.

File: src/docs/asciidoc/setting_up_clusters.adoc [hunk 2]
```diff
@@ -336,26 +336,26 @@ so that you can isolate your clusters.
...
 
-You can create cluster groups. To do this, use the `group` configuration
+You can create cluster. To do this, use the `cluster` configuration
 element.
```
Comment (Serdaro) [inline src/docs/asciidoc/setting_up_clusters.adoc]: You can create clusters using the `cluster` configuration element.

**Pull request merged**